### DISCUSSION TITLE
sqlite3.OperationalError: incomplete input

### DISCUSSION THREAD

**Discussion by LaumTop** [Rapptz/discord.py#8987] [Q&A] [answered]
conn = sqlite3.connect('Users.db')
    cursor = conn.cursor()
    conn.execute("""CREATE TABLE IF NOT EXISTS users(
                 username TEXT,
                 userid INT PRIMARY KEY,
                 KN TEXT)
                 """)

    cursor.execute(f"INSERT INTO users VALUES(username, userid, KN" "VALUES(?,?,?)",
                   ('name', 'id', 'KN'))

    conn.commit()

I have this code and I dont understand why it dont work, pls help
error: sqlite3.OperationalError: incomplete input

**Comment by JDJGInc** [comment]
Please join the discord
At discord.gg/dpy
The people here can help well.
If you are banned I am sure you could appeal

**Comment by DenverCoder1** [comment, marked as answer]
It seems like you're missing a closing parenthesis near

`VALUES(username, userid, KN`
